src/results.jl

```diff
@@ -0,0 +1 @@
+abstract type Results end
```

Comment:
I think this is too abstract.  We should define the behavior of this object.  This could be by defining interface functions or fields.